corona-warn-app/cwa-app-ios
# Feature/5409 check in info screen/5821 UI tests

## Description
UITests of trace locations:
- info screen 
- create QR code
- delete QR codes individually
- delete QR codes all together
- display details of trace location
- check-in 

UI tests of My Check-Ins:
- checkout
- display details of event
- display events in contact journal

Screenshots:
- tracelocation infoScreen
- tracelocation overview
- tracelocation detail_view
- tracelocation pdf_view
- mycheckins overview
- mycheckins details
- mycheckins allCheckedOut

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5581

## Screenshots
